tasks/src/object-detection/data.ts

```diff
@@ -1,13 +1,17 @@
 import type { TaskDataCustom } from "../Types";
 
 const taskData: TaskDataCustom = {
-	datasets: [
-		{
-			// TODO write proper description
-			description: "Widely used benchmark dataset for multiple Vision tasks.",
-			id:          "merve/coco2017",
-		},
+		datasets: [
+	  {
+		description: "Widely used benchmark dataset for multiple Vision tasks.",
+		id: "merve/coco2017",
```

Comment:
I think there are many generic object detection datasets on Hub compared to this :)